From 1b7bc027d26dbd8dd4e84d31d1c10e3fa67aec0a Mon Sep 17 00:00:00 2001
Message-ID: <1b7bc027d26dbd8dd4e84d31d1c10e3fa67aec0a.1721399836.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1721399836.git.sharlatanus@gmail.com>
References: <cover.1721399836.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Fri, 19 Jul 2024 15:28:01 +0100
Subject: [PATCH 6/7] gnu: go-github-com-mitchellh-mapstructure: Update to
 1.5.0.

* gnu/packages/golang.scm (go-github-com-mitchellh-mapstructure): Update to 1.5.0.

Change-Id: I3e18dabc4da5e4eec40cfe4328a6244dcc530e7d
---
 gnu/packages/golang.scm | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index 698dfe0120..fed47a3daf 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -4781,7 +4781,7 @@ (define-public go-github-com-mitchellh-go-homedir
 (define-public go-github-com-mitchellh-mapstructure
   (package
     (name "go-github-com-mitchellh-mapstructure")
-    (version "1.1.2") ;; NOTE: Updating to 1.3.1 breaks tests on viper-1.7.0
+    (version "1.5.0")
     (source
      (origin
        (method git-fetch)
@@ -4791,7 +4791,7 @@ (define-public go-github-com-mitchellh-mapstructure
        (file-name (git-file-name name version))
        (sha256
         (base32
-         "03bpv28jz9zhn4947saqwi328ydj7f6g6pf1m2d4m5zdh5jlfkrr"))))
+         "10f2v143lkip8h46shd99k5yavfqpgqmd7a6y42v7szc0lcn3mff"))))
     (build-system go-build-system)
     (arguments
      `(#:import-path "github.com/mitchellh/mapstructure"))
-- 
2.41.0

